docker/docker-compose.yml

```diff
@@ -22,6 +22,8 @@ services:
     networks:
       - stb-backend
       - stb-frontend
+    # @todo #236:60m Remove app service's depenency from a nodejs servise.
```

Comment:
`depenency` -> `dependency`
`servise` -> `service`.

Add plz `se2` at the end of title

`30 mins` will be enough for this task, i think. It's only about volumes